qiskit_experiments/library/characterization/ramsey_xy.py

```diff
@@ -208,3 +208,278 @@ def _metadata(self):
+
+        .. parsed-literal::
+
+            (Ramsey X)  The pulse before measurement rotates by pi-half around the X axis
+
+                     ┌────┐┌────────┐┌───┐┌───────────────┐┌────────┐┌────┐┌─┐
+                  q: ┤ √X ├┤ StarkV ├┤ X ├┤ StarkU(delay) ├┤ Rz(-π) ├┤ √X ├┤M├
+                     └────┘└────────┘└───┘└───────────────┘└────────┘└────┘└╥┘
+                c: 1/═══════════════════════════════════════════════════════╩═
+                                                                            0
+
+            (Ramsey Y) The pulse before measurement rotates by pi-half around the Y axis
+
+                     ┌────┐┌────────┐┌───┐┌───────────────┐┌───────────┐┌────┐┌─┐
+                  q: ┤ √X ├┤ StarkV ├┤ X ├┤ StarkU(delay) ├┤ Rz(-3π/2) ├┤ √X ├┤M├
+                     └────┘└────────┘└───┘└───────────────┘└───────────┘└────┘└╥┘
+                c: 1/══════════════════════════════════════════════════════════╩═
+                                                                               0
+
+        In principle, the sequence is a variant of :class:`.RamseyXY` circuit.
+        However, the delay in between √X gates is replaced with the off-resonant drive.
+        This off-resonant drive shifts the qubit frequency due to the
+        Stark effect and causes it to accumulate phase during the
+        Ramsey sequence. This frequency shift is a function of the
+        offset of the Stark tone frequency from the qubit frequency
+        and of the magnitude of the tone.
+
+        Note that the Stark tone pulse (StarkU) takes the form of a flat-topped Gaussian envelope.
+        The magnitude of the pulse varies in time during its rising and falling edges.
+        It is difficult to characterize the net phase accumulation of the qubit during the
+        edges of the pulse when the frequency shift is varying with the pulse amplitude.
+        In order to simplify the analysis, an additional pulse (StarkV)
+        involving only the edges of StarkU is added to the sequence.
+        The sign of the phase accumulation is inverted for StarkV relative to that of StarkU
+        by inserting an X gate in between the two pulses.
+
+        This technique allows the experiment to accumulate only the net phase
+        during the flat-top part of the StarkU pulse with constant magnitude.
+
+    # section: analysis_ref
+        :py:class:`RamseyXYAnalysis`
+
+    # section: see_also
+        qiskit_experiments.library.characterization.ramsey_xy.RamseyXY
+
+    """
+
+    def __init__(
+        self,
+        physical_qubits: Sequence[int],
+        backend: Optional[Backend] = None,
+        **experiment_options,
+    ):
+        """Create new experiment.
+
+        Args:
+            physical_qubits: Index of physical qubit.
+            backend: Optional, the backend to run the experiment on.
+            experiment_options: Extra experiment options. See ``self.experiment_options``.
```

Comment:
I am not sure that referencing `self.experiment_options` helps that much. Maybe something like this?

```suggestion
            experiment_options: Experiment options. See the class documentation or ``self._default_experiment_options`` for descriptions.
```